corpus/pages/views.py

```diff
@@ -42,5 +46,40 @@ def sig(request, sig_name):
 
     return render(request, "pages/sig.html", args)
 
+
+def team(request):
+    compsoc_members = ExecutiveMember.objects.filter(
+        (Q(core__isnull=True) | Q(core=None))
+        & (Q(user__faculty__isnull=True) | Q(user__faculty=None))
```

Comment:
`User` does not have `faculty`

Reply:
No but faculty can have a user, right? So I'm just checking that the user I'm fetching does not have a faculty linked to it